src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1395,11 +1395,23 @@
                 "Ubuntu Mono"
             ]
         },
-        "python_default_interpreter": {
+        "python_type": {
+            "title": "Python Type",
+            "description": "The Python type.",
             "type": "string",
-            "title": "Default Python interpreter",
-            "default": "",
-            "description": "The path to the default Python interpreter"
+            "default": ""
+        },
+        "python_version": {
+            "title": "Python Version",
+            "description": "The Python version.",
+            "type": "string",
+            "default": ""
+        },
+        "python_path": {
+            "title": "Python Path",
```

Comment:
The `title` is only used in the Command Palette, which doesn't show options w/o a Title -- so by convention options that can only be correctly manipulated by dedicated UI have no title. In this case I think we should remove the title so these options don't show up in the palette.